src/Files.App/Services/Settings/GeneralSettingsService.cs

```diff
@@ -263,6 +263,12 @@ public FileNameConflictResolveOptionType ConflictsResolveOption
 			set => Set(value);
 		}
 
+		public bool DisplaySettingsButtonOnSidebar
```

Comment:
Can you add to `RaiseOnSettingChangedEvent`?

Reply:
https://github.com/files-community/Files/pull/15458#discussion_r1617830808